src/pages/index.tsx

```diff
@@ -1,18 +1,18 @@
 import Image from "next/image";
 import { Inter } from "next/font/google";
-import Navbar from "./navabar";
+import Navbar from "../components/navabar";
```

Comment:
Hi @jakemelvin  at the root of your component folder do include an index file that exports all your components . like this

```
import ComponentOne from './ComponentOne';
import ComponentTwo from './ComponentTwo';

export {
  ComponentOne,
  ComponentTwo,
};
```

It will improve your imports at top of the file  and will make it look clean.

Reply:
OKay thanks